docs/RFCS/20191202_cluster_backup_restore.md

```diff
@@ -1,4 +1,4 @@
-- Feature Name: Full Cluster Backup/Restore
+- Feature Name: Cluster Backup/Restore*
```

Comment:
This is a _very_ weak preference and I can easily be convinced otherwise or just be ignored quite happily, but i'm lukewarm on renaming the RFC:  I think in naming the project and the technical design, the name made and makes sense since it emphasizes the design goals as long as the user-facing feature it describes is just a "cluster backup".

Reply:
FWIW, a search for "cluster backup"  would still find "full cluster backup" so I'm not particularly worried about the discoverability side.